requirements.dev.txt

```
@@ -1,3 +1,5 @@
+-r requirements.txt
```

Can you use the long-option to match what's in the README?
I tend to prefer long-options (if available) in scripts because they can be self-documenting.

```suggestion
--requirement requirements.txt
```

---

:+1: